php/storage/app/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
what's the point of the folder storage? it doesn't seem to do anything and there is nothing inside its folder/subfolders, is it required for PHP to work?

Reply:
Lumen populates them when running, this folder structure is a lumen default:
- https://github.com/laravel/lumen/tree/8.x/storage

Reply:
Removing the `storage/` directory causes instability. So it's needed or I'd have to configure a separate path.

![Cursor_and_Please_provide_a_valid_cache_path___500_Internal_Server_Error_](https://user-images.githubusercontent.com/956479/133785756-aca5fd61-e925-4353-bf48-c0e43d4722e5.png)